src/Calendar.js

```diff
@@ -210,6 +210,11 @@ let Calendar = React.createClass({
     step: React.PropTypes.number,
 
     /**
+     * Show all day events
+     */
+    showAllDayEvents: React.PropTypes.bool,
```

Comment:
I don't love this option because it will cause all and multi day events to just disappear in Week or Day views. Can you talk a bit why you'd want that?